apps/medon-fe/src/app/app.tsx

```diff
@@ -6,6 +6,9 @@ import ResetPassword from 'pages/ResetPassword';
 import ResendConfirmation from 'pages/ResendConfirmation';
 import UpdatePassword from 'pages/UpdatePassword';
 import Navigation from 'components/Navigation/index';
+import { PatientsPage } from 'pages/PatientsPage';
```

Comment:
components and pages shouldn't be exported as default?

Reply:
imported, sorry

Reply:
I try don't use default imports when it's possible. Because this is implicit way for import, you don't know exactly what you are importing, so it can leads issues. We should ask our mentor:)

Reply:
import ResendConfirmation from 'pages/ResendConfirmation';
import UpdatePassword from 'pages/UpdatePassword';
import { PatientsPage } from 'pages/PatientsPage';

import Navigation from 'components/Navigation/index';

sorry, but its so paintfull see the mess